src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        System.out.println(ball1.toString());
+        System.out.println(ball2.toString());
+        System.out.println(ball3.toString());
```

Comment:
<img width="1022" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/81781479/c57f3a11-5613-40a9-817e-fccdea3bdaa1">

Reply:
I've fixed this one